backend/src/routes/api/k8s/index.ts

```diff
@@ -1,9 +1,17 @@
 import { FastifyReply, FastifyRequest } from 'fastify';
 import { KubeFastifyInstance } from '../../../types';
 import { USER_ACCESS_TOKEN } from '../../../utils/constants';
-import { passThrough } from './pass-through';
+import { passThrough, safeURLPassThrough } from './pass-through';
 import { logRequestDetails } from '../../../utils/fileUtils';
 
+// TODO: Remove when bug is fixed - https://issues.redhat.com/browse/HAC-1825
+// Some resources don't allow metadata.name when creating such as SelfSubjectAccessReview
+// this will cause the URL error when we call pass-through function
+// we will check this value below to call safeURLPassThrough instead of passThrough
+enum resourceWithoutMetadataName {
```

Comment:
Don't know if this is the best way to do it, waiting for some suggestions. cc @andrewballantyne @Gkrumbach07 @lucferbux @alexcreasy 
Because Core SDK builds the wrong path for k8s when making a POST call, so we did some URL processes in the passThrough method. However, it doesn't work for `SelfSubjectAccessReview`, when creating the resource, at first I didn't set `metadata.name`, and the endpoint is `.../api/k8s/apis/authorization.k8s.io/v1/selfsubjectaccessreviews` because the name is set, and after processing it becomes `.../api/k8s/apis/authorization.k8s.io/v1`. After that, I tried to create the `SelfSubjectAccessReview` and set the `metadata.name` for it. The URL is right but the server returns me a 422 unprocessable error and tells me the name should be empty (which means I couldn't set metadata.name when creating the `SelfSubjectAccessReview` resource).
Finally, I come up with this method, which creates a list including all the resources that cannot have a `metadata.name` when creating and checks it below to decide whether to use `safeURLPassThrough` or `passThrough`.